src/components/layout/Footer.tsx

```diff
@@ -119,11 +119,6 @@ export const Footer: FunctionComponent<FooterProps> = ({ mobile }) => {
             {t("navigation.wiki-label")}
           </FooterTextLink>
         </li>
-        <li>
-          <FooterTextLink href={links.LEGACY_BRIDGE} target="_blank">
-            {t("navigation.legacy-bridge-label")}
```

Comment:
if unused in other places, remove from `en.json` (just this file).

Reply:
Thanks, I was going to ask how that works. Will Crowdin remove the string from other languages?